steps/http_request.md

```diff
@@ -35,7 +35,18 @@ have query parameters that can be read like this:
 
 ### req.body
 
-To read parameters send from HTML forms one use the `req.body` paramater on the Http Request object sent into routes.
+To read parameters sent from HTML forms to the server you use the `body` attribute on the HttpRequest object sent into routes.
+
+For a html form that looks like this in html:
+
+```html
+<form action="/add_product" method="POST">
+    Product name : <input type="text" name="product_name" >
```

Comment:
`<label>Product name: <input type="text" name="product_name" ></label>`

Not sure whether to do that or 

`<label for="product_name">Product name:</label> <input type="text" name="product_name" id="product_name">`

I think the first one.